plugins/zfs/zfs_pool_dataset_count

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#%# family=auto
+
+: << EOF
+=head1 NAME
+zfs_pool_dataset_count - Outputs the count of zfs pools, datasets and snapshots.
+
+
+=head1 AUTHOR
+=over 4
+=item * Michael Grote
+=back
+
+=head1 LICENSE
+Unknown license
+
+=head1 MAGIC MARKERS
+=begin comment
+These magic markers are used by munin-node-configure when installing
+munin-node.
+=end comment
+  #%# family=auto
+  #%# capabilities=autoconf suggest
```

Comment:
Please remove `suggest`, since this is not a wildcard plugin.

Reply:
removed